gp-nested-forms/gpnf-child-entry-id-modifier.php

```diff
@@ -19,7 +19,7 @@
 	}
 
 	$dom = new DOMDocument();
-	$dom->loadHTML( $markup );
+	$dom->loadHTML( '<?xml encoding="utf-8" ?>' . $markup );
```

Comment:
This is an XML declaration which might cause some unforeseen issues. Before pressing on with this, can you give this a try?

Change `$dom = new DOMDocument();` to `$dom = new DomDocument('1.0', 'UTF-8');`

Either way, nice work finding the issue!